### DISCUSSION TITLE
What does "LXXXVIII" mean in Arabic numerals?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#784] [Galaxy Trivia] [answered]
**Bonus:** What would be the Roman numerals for today's DOY?

**Comment by SANKARAMDAS** [comment]
The Roman numeral LXXXVII corresponds to the Arabic number 87

V • XX • MMXXIII

**Comment by peterbaccaro** [comment, marked as answer]
"LXXXVIII" represents the number **88** in Arabic numerals. In the Roman numeral system, "L" stands for 50, "XXX" stands for 30, and "VIII" stands for 8. When combined, they represent the value of 50 + 30 + 8, which is equal to 88.

Today is May 20th, 2023 so the day of the year (DOY) would be 140 (May 20th is the 140th day of the year in a non-leap year).
To represent 140 in Roman numerals, we can break it down as follows:
- "C" represents 100
- "XL" represents 40 (50 - 10)
- "X" represents 10
Combining these values, we can represent 140 as "**CXL**".
